src/traces/sankey/render.js

```diff
@@ -60,12 +60,29 @@ function sankeyModel(layout, d, traceIndex) {
       .nodes(nodes)
       .links(links);
 
-    var graph = sankey();
-
     if(sankey.nodePadding() < nodePad) {
         Lib.warn('node.pad was reduced to ', sankey.nodePadding(), ' to fit within the figure.');
     }
 
+    var graph = sankey();
+    Object.keys(calcData._groupLookup).forEach(function(nodePointNumber) {
+        var groupIndex = parseInt(calcData._groupLookup[nodePointNumber]);
+        var groupingNode = graph.nodes.find(function(node) {
+            return node.pointNumber === groupIndex;
+        });
+        groupingNode.key = groupingNode.key + ',' + nodePointNumber;
+        graph.nodes.push({
+            pointNumber: parseInt(nodePointNumber),
+            x0: groupingNode.x0,
+            x1: groupingNode.x1,
+            y0: groupingNode.y0,
+            y1: groupingNode.y1,
+            partOfGroup: true,
```

Comment:
`partOfGroup === true` means that this node is included in another node and is a children. It looks as if it's unliked and was deleted by the `d3-sankey*` engines.